### PR TITLE
feat: Updating minimum supported node version

### PR DESCRIPTION
**Pull request by gvsvarun**
Also updated the current node version blog as well

Fixes #{issue}

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by gvsvarun** [comment]
https://github.com/jwu910/check-it-out/issues/397

**Comment by jwu910** [comment]
@gvsvarun 	Thanks for taking a look into this and sending a PR. However, there is a change we'll have to make on your PR. The minimum node version we are supporting should be Node v6.0.0. The link should not point to node 12 as that would be misleading.

It seems that there was a mistake already and we were pointing to node 7 and were still displaying node 4 as a requirement.

File: index.html [hunk 1]
```diff
@@ -71,7 +71,7 @@ <h2> What is Check It Out? </h2>
 		<div class="require-section section">
 			<h2> Requirements </h2>
 			<p>
-			<a href="https://nodejs.org/en/blog/release/v7.0.0/">Node >= v4.0 </a>
+			<a href="https://nodejs.org/en/blog/release/v12.3.1/">Node >= v6.0 </a>
```
Comment (jwu910) [inline index.html]: We should be pointing to node v6.0.0

Comment (gvsvarun) [reply index.html]: Sounds logical. Will fix and update.

**Comment by jwu910** [comment]
Thanks, merged!

**Pull request merged**